.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
At present time, I don't want to check-in editor-specific files into the project.  I realize that you added this as a result of needing to undo your other formatting changes , and I'm open to being convinced otherwise, but at the moment I'm not.

Reply:
ah ...how do i remove the file...just add it to gitignore?

Reply:
1. delete the file
2. `git add .vscode/settings.json`
3. `git commit`
4. `git push`

Reply:
thanks...this is done